app/services/reports/region_service.rb

```diff
@@ -11,11 +11,11 @@ def self.call(*args)
       new(*args).call
     end
 
-    def initialize(region:, period:, with_exclusions: false)
+    def initialize(region:, period:, with_exclusions: false, total_months_requested: MAX_MONTHS_OF_DATA)
```

Comment:
Suggestion: a name like `number_of_months` or just `months_request` would be a bit more clear and concise

Reply:
Will do. I was feeling pretty ambivalent about the naming.